docs/installation.rst

```diff
@@ -0,0 +1,11 @@
+Installation
+==================
+
+SublimeLinter and corresponding linter plugins should be installed using `PackageControl <https://packagecontrol.io/installation>`_.
+
+First install PackageControl and then see its `usage <https://packagecontrol.io/docs/usage>`_,
+which explains how to install plugins to SublimeText.
+
+Notice, that it may be required to restart SublimeText after plugin installation.
+
+Make sure to read `SublimeLinter messages <https://github.com/SublimeLinter/SublimeLinter/tree/master/messages>`_.
```

Comment:
This reference seems odd. Messages are shown on updates, so users should rather read the messages that apply to them in their editor when they open.

Reply:
Yes, but looking at the amount of issues on GitHub it seems people are not reading those.
Also AFAIR it shows only last change, not sure if it shows all messages between current and freshly installed versions....